### PR TITLE
update the documents of podTopologySpread

### PR DESCRIPTION
**Pull request by kerthcet**
Signed-off-by: kerthcet <kerthcet@gmail.com>

Update the documents of podTopologySpread

### CODE REVIEW

**Review by sftim — COMMENTED** [review]
A couple of tweaks and a question. This is ready for tech review (SIG Scheduling, I'd assume).

File: content/en/docs/concepts/scheduling-eviction/topology-spread-constraints.md [hunk 1]
```diff
@@ -556,7 +561,7 @@ section of the enhancement proposal about Pod topology spread constraints.
   cluster. This could lead to a problem in autoscaled clusters, when a node pool (or
   node group) is scaled to zero nodes, and you're expecting the cluster to scale up,
   because, in this case, those topology domains won't be considered until there is
-  at least one node in them.
```
[the remark this answers is not in the document]
Comment (kerthcet) [reply content/en/docs/concepts/scheduling-eviction/topology-spread-constraints.md:559]: Updated. The two spaces were removed by the formatter.

**Comment by kerthcet** [comment]
/assign @alculquicondor
It's an update of current podTopologySpread documents.

File: content/en/docs/concepts/scheduling-eviction/topology-spread-constraints.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (kerthcet) [reply content/en/docs/concepts/scheduling-eviction/topology-spread-constraints.md:559]: Added a actual new line

**Review by alculquicondor — COMMENTED** [review]
/lgtm

**Pull request merged**